bedrock/firefox/templates/firefox/geolocation.html

```diff
@@ -201,6 +201,20 @@ <h3 data-accordion-role="tab">{{ _('How do I turn off Location-Aware Browsing pe
   </div>
 </section>
 
+<section>
+  {% if l10n_has_tag('geolocation_faq_update') %}
+    <h3 data-accordion-role="tab">{{ _('How can I add support for Geolocation to my web site?') }}</h3>
+    <div data-accordion-role="tabpanel">
+      <p>
+      {% trans url="https://developer.mozilla.org/docs/Web/API/Geolocation/Using_geolocation" %}
+      You can integrate Geolocation support into your service by following the instructions
+      <a href="{{ url }}">Mozilla Developer Center</a>.
```

Comment:
Nit: still missing an `at` before `Mozilla Developer Center`

It should read: `You can integrate Geolocation support into your service by following the instructions at Mozilla Developer Center`